packages/dds/cell/src/test/cell.spec.ts

```diff
@@ -141,6 +143,22 @@ describe("Cell", () => {
                 assert.equal(cell2.get(), newValue, "The second cell did not get the new value");
             });
         });
+
+        describe("Summarization of the Attribution", () => {
+            it("Should sync with the attribution", async () => {
+                // overwrite the cell with attribution tracking enabled
+                cell = createLocalCell("cell", true);
+                cell.set("value");
+
+                // load a cell from the snapshot
+                const services = MockSharedObjectServices.createFromSummary(cell.getAttachSummary().summary);
+                const cell2 = new SharedCell("cell2", new MockFluidDataStoreRuntime(), CellFactory.Attributes);
+                await cell2.load(services);
+
+                // Verify the attribution of SharedCell with all pending messages processed
+                assert.equal(cell.getAttribution()?.seq, cell2.getAttribution()?.seq, "the attribution key should be consistent");
```

Comment:
i'd validate the result you get here isn't undefined as well. I'd prefer the test fails if both cells have undefined attribution (even if you could argue that the round-tripping is correct, i'd rather have the test fail because it's not testing what it intended to in that case)